jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/editor/CodeWhispererEditorUtil.kt

```diff
@@ -90,6 +92,17 @@ object CodeWhispererEditorUtil {
         )
     }
 
+    fun shouldSkipInvokingBasedOnRightContext(editor: Editor): Boolean {
+        val caretContext = runReadAction { CodeWhispererEditorUtil.extractCaretContext(editor) }
+        val rightContextLines = caretContext.rightFileContext.split(Regex("\r?\n"))
+        val rightContextCurrentLine = rightContextLines[0]
```

Comment:
might worth wrapping it with `tryOrNull {}` in case the array is empty

Reply:
updated